src/components/Details/BootOrder/BootOrder.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const BootOrder = props => {
+  const { bootableDevices } = props;
+  const sortedBootableDevices = bootableDevices.sort((a, b) => a.bootOrder - b.bootOrder);
+  const listItems = [];
```

Comment:
can be reduced to `listItems = sortedBootableDevices.map((dev, idx) => <li key={`${idx}-${dev.name}`}>{dev.name}</li>)`;

Reply:
Nice and clean! Done!